content/departments/engineering/dev/process/releases/release_issue_template.md

```diff
@@ -108,8 +110,12 @@ On the day of the release, confirm there are no more release-blocking issues (as
 
 ### Post-release
 
-- [ ] Notify the next [release captain](./index.md#release-captain) that they are on duty for the next release.
```

Comment:
This link is broken, but using an absolute link failed on lint, i'd rather just remove it.